llm/gpt-2/gpt2-train.yaml

```diff
@@ -0,0 +1,89 @@
+name: train
+
+envs:
+  BUCKET_NAME: # Fill in your bucket name
+
+resources:
+  accelerators: A100:8
+  # Use docker image for latest version g++ to enable the compilation of llm.c.
+  image_id: docker:nvidia/cuda:12.4.1-cudnn-devel-ubuntu22.04
+  image_id:
```

Comment:
YAML syntax seems to be invalid.. I'm getting:
```
ValueError: Invalid resources YAML: [{'cloud': 'lambda', 'image_id': None}, {'cloud': 'aws'}, {'cloud': 'gcp'}, {'cloud': 'azure'}, {'cloud': 'fluidstack'}] is not valid under any of the given schemas. Check problematic field(s): $.image_id
```

Should this be `any_of`? with `any_of` I get `ValueError: Invalid resources YAML: None is not valid under any of the given schemas. Check problematic field(s): $.any_of[0].image_id`.